ihmc-perception/src/main/java/us/ihmc/perception/RawImage.java

```diff
@@ -224,11 +267,26 @@ public GpuMat getGpuImageMat()
       return gpuImageMat;
    }
 
+   public BytePointer getDataPointer()
+   {
+      return getCpuImageMat().data();
+   }
+
+   public BytePointer getCUDADataPointer()
```

Comment:
Maybe add a comment that because this calls getGpuImageMat, it allocates CUDA memory if it didn't exist